external/loaders/loaders/mappers/_fine_res.py

```diff
@@ -63,11 +69,10 @@ def _open_merged_dataset(
     else:
         merged = fine
 
-    # enforce that these ML inputs come from fine dataset if they exist there
-    if "T" in fine:
-        merged["air_temperature"] = fine.T
-    if "sphum" in fine:
-        merged["specific_humidity"] = fine.sphum
+    if use_fine_res_state:
+        # overwrite standard name arrays with those from fine-res budget
+        for fine_res_name, standard_name in FINE_RES_BUDGET_NAMES.items():
+            merged[standard_name] = fine[fine_res_name]
```

Comment:
Would dropping the (duplicated) fine-res named variables when `use_fine_res_state` is true help simplify things?

Reply:
Yeah, I was considering that. The benefit of not dropping is that then we can always assume that the GFDL-style names will always be available in the mapper output, no matter what the `use_fine_res_state` flag is set to. Do you have a strong preference?

Reply:
Up to you

Reply:
Okay I'm going to leave as is. I don't think there'll be a performance cost, since these will just be references to the other arrays.